packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -543,6 +542,11 @@ function UserListTableContent() {
     }
   };
 
+  if (!isSuccessAttributes) {
```

Comment:
this is to avoid flickering (attributes are fetched at the similar timing as the table data, anyway)